src/main/java/com/es/phoneshop/model/product/Product.java

```diff
@@ -12,7 +13,7 @@
 @Getter
 @Setter
 @EqualsAndHashCode
-public class Product {
+public class Product implements Entity {
```

Comment:
why implements
and where is Serializable

Reply:
I decided to make Entity an interface , because everything we need from entity in our dao (basic interface) is to get entity id, and it allows us to make any class an entity , even if it extends another class (like in case with order that extends cart)